lib/dry/logic/predicates.rb

```diff
@@ -134,7 +134,15 @@ def self.included(other)
       end
 
       predicate(:exclusion?) do |list, input|
-        !self[:inclusion?].(list, input)
+        !list.include?(input)
```

Comment:
Something @solnic said about my not_eql predicate.

>> you may consider just doing !left.eql(right) instead of negating the eql predicate, tiny perf improvement.